install_config/configuring_authentication.adoc

```diff
@@ -200,6 +200,49 @@ identity providers are configured that identify the same set of users and map to
 the same user names.
 |===
 
+[[LookupMappingMethod]]
+==== Lookup Mapping Method
```

Comment:
I think the title could be more descriptive. Maybe "Manually mapping an identity using the Lookup Mapping Method" or similar.